gnomad_qc/v4/sample_qc/create_sample_qc_metadata_ht_genomes.py

```diff
@@ -1,10 +1,20 @@
 """
 Script to create sample QC metadata HT for genomes.
 
-This script is used to create the sample QC metadata HT for genomes. The main updates
-from v3.1 to v4.0 are:
-Sample QC annotations for HGDP + TGP subset, this script directly merges the updated
-annotations from the subset meta HT to the full meta HT.
+The main updates from v3.1 to v4.0 are due to the following updates in the HGDP + TGP subset meta:
+- 169 samples added
+- 110 samples removed
+
+The 169 samples added were diverse HGDP/TGP samples in v3.1 that were unintentionally removed during sample outlier detection.
+
+The 110 samples were removed due to the following reasons:
+- Contamination flagged by CHARR.
+- Subcontinental PCA outliers within the subset.
+- Updated relatedness results within the subset, re-evaluation of relatedness
+  inference of the subset to the rest of gnomAD release samples, and removal of any
+  that are duplicates of v4.0 exomes release samples.
```

Comment:
```suggestion
  samples that are duplicates of v4.0 exomes release samples.
```
also basic question: there were duplicates between the HGDP/1KG samples and v4 samples?

Reply:
Yes, there are a few: https://github.com/broadinstitute/gnomad_qc/blob/8e55307cfef330371174f03b40089a3975df7e00/gnomad_qc/v4/annotations/generate_freq_genomes.py#L190

Reply:
follow-up question: why weren't these caught by cuKING? also, this means that our documentation
```
After running cuKING, we used the [maximal_independent_set method in Hail](https://hail.is/docs/0.2/methods/misc.html#hail.methods.maximal_independent_set) to get the largest set of samples with no pair of samples related at the 2nd degree or closer. When we had to select a sample amongst multiple possibilities, we used the same scheme to favor a sample over another:
Genomes had priority over exomes in gnomAD v4.0
Within genomes
- PCR free had priority over PCR+
- In a parent/child pair, the parent was kept
- Ties broken by the sample with highest mean coverage
Within exomes
- In a parent/child pair, the parent was kept
- Ties broken by the sample with the higher mean coverage on chromosome 20
```
is no longer fully correct, right?

Reply:
It's caught in cuKing, that's the results we're using to get them. Oh, I think here we put 'exomes' as priority.. @jkgoodrich could you confirm this?

Reply:
I meant we didn't want to change the number of release samples in exomes, this one came after, almost the last step when I realized we should check the duplicates.

Reply:
follow-up question: if these samples were flagged by cuKING as being duplicates, wouldn't the v4 exomes results have been filtered such that this wouldn't be an issue? as in, shouldn't the HGDP/1KG samples been prioritized in the pruning given the logic here: https://github.com/broadinstitute/gnomad_qc/blob/main/gnomad_qc/v4/sample_qc/relatedness.py#L275? or were these samples not releasable in v3 to begin with?

Reply:
Yes, because these samples weren't in v3 release, so they weren't considered before. Now I used them as in v3 (not in v3 release) to get the duplicates.

Reply:
hmmm. OK. I think this is fine then, and consistent with our documentation